docs/RELEASE-NOTES.rst

```diff
@@ -1,11 +1,13 @@
 Release Notes for K8S BIG-IP Controller
 ============================================
 
-v1.1.0-dev
+v1.1.0
 ----------
```

Comment:
Please undo this. The master branch will have the '-dev' suffix. The release branch will strip it.